src/controllers/addressBook/addressBook.ts

```diff
@@ -0,0 +1,160 @@
+import { getAddress } from 'ethers'
+
+import { Storage } from '../../interfaces/storage'
+import EventEmitter from '../eventEmitter/eventEmitter'
+
+export type Contacts = Array<{
+  name: string
+  address: string
+  isWalletAccount?: boolean
```

Comment:
TBD: It might make sense to store the `createdAt` and the `updatedAt` timestamps for each Contact. It won't hurt and having this additional meta information could help us in the future.